python/interpret_community/permutation/permutation_importance.py

```diff
@@ -243,7 +244,10 @@ def __init__(self, model, is_function=False, metric=None, metric_args=None, is_e
                 model = WrappedPytorchModel(model)
         except (NameError, AttributeError):
             log_pytorch_missing = True
-        super(PFIExplainer, self).__init__(model, is_function=is_function, **kwargs)
+        if is_sequential(model):
```

Comment:
The test is in a different repo (currently disabled).  Similar test exists in interpret-community, but since this issue only appears with tensorflow>2.6 (currently we pin to older version of tensorflow due to DeepExplainer), we don't see this test failure in interpret-community yet.  However, this code path is tested...